packages/application-generic/src/services/schedulers/providers/agenda.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Agenda,
+  IDatabaseOptions,
+  IDbConfig,
+  IMongoOptions,
+} from '@hokify/agenda';
+
+export type AgendaFactoryOptions = {
+  name?: string;
```

Comment:
i'm currently not setting any configs except DB url while instantiating agenda, let me know if you think it will be useful to make them env variables and use a default value on explicitly.